libraries/AP_Airspeed/AP_Airspeed_External.h

```diff
@@ -0,0 +1,38 @@
+/*
+  external AHRS airspeed backend
+ */
+#pragma once
+
+#include "AP_Airspeed_config.h"
+
+#if AP_AIRSPEED_EXTERNAL_ENABLED
+
+#include "AP_Airspeed_Backend.h"
+#include <AP_ExternalAHRS/AP_ExternalAHRS.h>
+
+class AP_Airspeed_External : public AP_Airspeed_Backend
+{
+public:
+    AP_Airspeed_External(AP_Airspeed &airspeed, uint8_t instance);
```

Comment:
```suggestion
    AP_Airspeed_External(AP_Airspeed &airspeed, const uint8_t instance);
```

Reply:
its not a reference, so why should the API for the function to not change it if it wants to? internally we could adjust for 0 vs 1 based for example. We don't do that in this case, but why should we make the interface tie this down?

Reply:
Fair enough. If I was a dev and the interface was locked down and I wanted to change the value, I'd just create a non-const copy and mutate it as needed.